inc/class-cachify-memcached.php

```diff
@@ -140,9 +140,12 @@ public static function clear_cache() {
 	/**
 	 * Print the cache
 	 *
-	 * @since 2.0.7
+	 * @param bool  $sig_detail Show details in signature.
+	 * @param array $cache      Array of cache values.
+	 *
+	 * @since 2.0
```

Comment:
Why was the `.7` dropped?
Not that anybody really cares, but up to 2.0.4 the class did not even exist, so 2.0 is strictly speaking not correct. There is no history between 2.0.4 and 2.1.6 available. In a SemVer world 2.1 would make sense, but I’d guess 2.0.7 is plausible, too.

Reply:
Good point. I missed the knowledge of the past development here.